set.mm

```diff
@@ -81241,12 +81616,10 @@ C Fn ( ( S i^i dom F ) u. { z } ) ) $=
        function is said to be defined by "well-ordered recursion".  The purpose
        of these three theorems is to demonstrate the properties of ` F ` .  We
        begin by showing that ` F ` is a function over ` A ` .  (Contributed by
-       Scott Fenton, 22-Apr-2011.)  (Revised by Mario Carneiro,
-       26-Jun-2015.) $)
+       Scott Fenton, 22-Apr-2011.)  (Revised by Mario Carneiro, 26-Jun-2015.)
+       (Proof shortened by Scott Fenton, 18-Nov-2024.) $)
     wfr1 $p |- F Fn A $=
```

Comment:
wfr1 and wfr2: same comments regarding the inference form: is it worth it ? Actually, the deduction forms would be more efficiently applicable. (Of course, sometimes, the inference forms provide a minor shortening, but that does not make for a strong argument.)

Reply:
Again, I was just following previous usage. I've revised the relevant proofs now.